frontend/src/views/TeamProfile/TeamProfileEditPage.vue

```diff
@@ -112,11 +146,29 @@ export default {
       backend_errors: ""
     }
   },
+  computed: {
+    remainingCharactersTeamtName() {
+      const maxChars = 255
+      const nameLength = this.team.name?.length ?? 0
```

Comment:
typescritpではなく、javascriptなので?は不要かと
```suggestion
      const nameLength = this.team.name.length ?? 0
```